docs/CONTRIBUTING.md

```diff
@@ -156,6 +156,7 @@ Our lists provide a minimal set of metadata: titles, URLs, creators, platforms,
 
 - No invented titles. We try to take titles from the resources themselves; contributors are admonished not to invent titles or use them editorially if this can be avoided. An exception is for older works; if they are primarily of historical interest, a year in parentheses appended to the title helps users know if they are of interest.
 - No ALLCAPS titles. Usually title case is appropriate, but when doubt use the capitalization from the source
+- No emoji or "weird" characters. The reading must be uniform.
```

Comment:
You could say "Not any special characters or emojis are allowed! For the purpose of readability just use the plain text."

Reply:
With "special characters" could mean characters like `|` or `&`, maybe you could include "(except  `|`, `&`...)"

Reply:
Yeah, it can be like this

Reply:
This is a book in our list: https://10print.org/

The title is "10 PRINT CHR$(205.5+RND(1)); : GOTO 10"

Reply:
Then we could simply write "No emoji" reserving the option of disallowing certain titles/characters if we found it necessary, outside of ordinary cases.

Reply:
The use case of weird in terms of character to some could actually mean  different alphabet systems, I suggest this change
```suggestion
- No emoji or nonstandard characters. The reading must be uniform.
```
Although this is still a little bit vague.

Reply:
I don't recall a case where anyone has submitted a listing containing an emoji or other "odd" character, unless you count `|`.  Let's not try to solve problems we don't have.  We frequently have problem with alphabetical order, for example, so maybe we can focus on that aspect?

Reply:
The "emoji problem" was on #6874 so I decided to add this rule.

Tomorrow I will commit a draft of the new section regarding alphabetical sorting so that we can discuss it.

Reply:
check the new commit